free_haaretz.js

```diff
@@ -22,6 +22,8 @@ chrome.webRequest.onBeforeSendHeaders.addListener(
   function (details) {
     for (var i = 0; i < details.requestHeaders.length; ++i) {
       if (details.requestHeaders[i].name.toLowerCase() === 'user-agent') {
+        if (typeof window.orientation !== 'undefined')
```

Comment:
It is. Ain't my idea thought, just found it on the internet and it seemed to be logical
Another option is to find out whether the user agent string contains the word 'mobile':
`if (navigator.userAgent.toLowerCase().indexOf('mobile') > -1) {...}`